File: C_Programming/Practice/program6.c
// Program to check whether number is even or odd

#include<stdio.h>

int CheckEvenOdd(int iNo)
{
    int iRemender = 0;

    iRemender = iNo % 2 ;

    return iRemender;

}

int main()
{
    int iValue = 0;
    int iRet = 0;
    
    printf("Enter number : ");
    scanf("%d",&iValue);

    iRet = CheckEvenOdd(iValue);

    if(iRet == 0)
    {
        printf("Number is even");
    }
    else
    {
        printf("Number is odd");
    }


    return 0;
}

File: Java_Programming/Practice/program6.java
// Check whether number is even or odd

import java.util.Scanner;

class program6
{
    public static void main(String A[])
    {
        int iValue = 0;
        int iRemainder = 0;

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter number : ");
        iValue = sc.nextInt();

        iRemainder = iValue % 2;

        if(iRemainder == 0)
        {
            System.out.println("Number is Even");
        }

        else
        {
            System.out.println("Number is Odd");
        }

    }

}